src/routes/controllers/articles.controller.js

```diff
@@ -47,11 +52,15 @@ export async function createArticle(req, res) {
       description,
       body,
       imageList: images,
-      tagList: tags,
+      tags: tagList,
       readTime: timeToReadArticle,
       subscriptionType: 'free',
       status: 'draft',
     });
+    const createdTag = await createTag(tagList, createdArticle.id);
+    if (!Array.isArray(createdTag)) {
+      return errorResponse(res, 201, createdTag);
```

Comment:
we should not return a success status code for errors.

Reply:
I have fixed it.